src/openzeppelin/tests/access/ownable/test_dual_ownable.cairo

```diff
@@ -0,0 +1,203 @@
+use traits::Into;
+use array::ArrayTrait;
+use starknet::ContractAddress;
+use starknet::contract_address_const;
+use starknet::testing::set_caller_address;
+use starknet::testing::set_contract_address;
+use zeroable::Zeroable;
+
+// Dispatchers
+use openzeppelin::access::ownable::interface::IOwnableDispatcher;
+use openzeppelin::access::ownable::interface::IOwnableCamelDispatcher;
+use openzeppelin::access::ownable::interface::IOwnableDispatcherTrait;
+use openzeppelin::access::ownable::interface::IOwnableCamelDispatcherTrait;
+
+// Dual case Ownable
```

Comment:
```suggestion
// DualCase Ownable
```

```suggestion
// Dual case ownable
```

```suggestion
// Dual case
```

i like any of these better, or even without since it's not really adding much information other than slightly grouping

Reply:
Fixed!